app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/CreateServiceRequest.php

```diff
@@ -67,7 +67,8 @@ public function form(Form $form): Form
                     ->relationship('division', 'name')
                     ->label('Division')
                     ->required()
-                    ->exists((new Division())->getTable(), 'id'),
+                    ->exists((new Division())->getTable(), 'id')
+                    ->default(auth()->user()->teams()->count() ? auth()->user()->teams[0]?->division?->id : ''),
```

Comment:
Can we use `->first()` instead of array access notation for better readability?

Reply:
> Can we use `->first()` instead of array access notation for better readability?

Hi,

That's been added.

Thank you.